package.json

```diff
@@ -153,6 +153,7 @@
     "i18next-http-backend": "1.4.4",
     "i18next-localstorage-backend": "3.1.3",
     "i18next-node-fs-backend": "2.1.3",
+    "is-extendable": "^1.0.1",
```

Comment:
seems no place use this, right? If no, pls remove this no use decency.

Reply:
This is need to get the project build successfully(npm run build).